pandas/io/gbq.py

```diff
@@ -27,6 +27,7 @@ def read_gbq(
     use_bqstorage_api=None,
     private_key=None,
     verbose=None,
+    progress_bar_type=None,
```

Comment:
```suggestion
    progress_bar_type: Optional[str] = None,
```

might as well put what you have in the doc as an annotation